### ISSUE TITLE
 https://lifehacker.com doesn't work in Sopel v6.5.0 with Python v2.7.

### ISSUE THREAD

**Issue by antdude** [sopel-irc/sopel#1226]
But worked fine on my 64-bit Debian Jessie/oldstable box with its Python v3.4.2 and Sopel (URL). Example from my friend's Sopel bot (Crushinator) with Python v2.7:
```
...
[12:24pm] <Ant>  http://lifehacker.com
12:24PM <URL> [ Lifehacker - Tips, tricks and downloads for getting things done ] - lifehacker.com
12:24PM <Crushinator> [ deadspin-quote-carrot-aligned-w-bgr-2<\/title><path d="M10,3.5l3-3,3,3Z"
          style="fill:%23fff;stroke:%23fff"/><path d="M0,3.5H10l3-3,3,3H26"
          style="fill:none;stroke:%231b3a4d"/><\/svg>')}.f_branding_on. ] - lifehacker.com
...
```

Thank you in advance. :)

**Comment by lenisko** [comment]
https://validator.w3.org/nu/?doc=https%3A%2F%2Flifehacker.com%2F I'm not surprised...

**Comment by dasu** [comment]
Did your friend try using this fix: #1215 ?
